src/Microsoft.Identity.Web/Resource/JwtBearerMiddlewareDiagnostics.cs

```diff
@@ -2,43 +2,50 @@
 // Licensed under the MIT License.
 
 using System;
-using System.Diagnostics;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Identity.Web.Resource
 {
     /// <summary>
     /// Diagnostics for the JwtBearer middleware (used in Web APIs)
     /// </summary>
-    public class JwtBearerMiddlewareDiagnostics
+    public class JwtBearerMiddlewareDiagnostics : IJwtBearerMiddlewareDiagnostics
     {
+        private readonly ILogger _logger;
+
+        public JwtBearerMiddlewareDiagnostics(ILogger<JwtBearerMiddlewareDiagnostics> logger)
+        {
+            _logger = logger;
+        }
+
         /// <summary>
         /// Invoked if exceptions are thrown during request processing. The exceptions will be re-thrown after this event unless suppressed.
         /// </summary>
-        private static Func<AuthenticationFailedContext, Task> s_onAuthenticationFailed;
+        private Func<AuthenticationFailedContext, Task> s_onAuthenticationFailed;
```

Comment:
Proposal to not use 's_' prefix for statics. Also these should use proper casing.

Reply:
that's something to bring up to the team. we have this as a rule in MSAL, so all the statics should shave this format.